meson.build

```diff
@@ -4,10 +4,9 @@ project(
   version : '1.8',
   meson_version : '>=0.51',
   default_options: [
-    'c_std=c99',
```

Comment:
We could set 'c_std=gnu11' to pin the standard version.

Reply:
In https://github.com/rauc/rauc/compare/4478e169d2ab51e18a47b58339496585e3a16dbe..5d566674bd024218bdd06bde7ab36a44ffa2b32b#diff-30d8f6be6320feeacf686be94f48c70869b52630e01ea625f0f15adc0d57c3e4L7, this was removed again?

Reply:
Oups, rebase fail. Fixed.